### PR TITLE
beautify tidb-operator-local-deployment.md

### PR DESCRIPTION
**Pull request by Jacklli**
beautify tidb-operator-local-deployment.md

### CODE REVIEW

**Review by cofyc — APPROVED** [review]
Great! This solved markdown lint issues.

File: session2/chapter1/tidb-operator-deployment-private-env.md [hunk 1]
```diff
@@ -1,14 +1,20 @@
 
-### 软件版本要求如下：
+# 集群环境资源需求
+
+## 软件版本要求如下
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-deployment-private-env.md]:
```suggestion
## 软件版本要求
```

File: session2/chapter1/tidb-operator-deployment-private-env.md [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+
 1. Docker CE 18.09.6
 2. Kubernetes v1.12.5+
 3. CentOS 7.6 以上，内核要求为 3.10.0-957 或之后版本
 
-### 部署 Cloud TiDB 资源要求如下:
+## 部署 Cloud TiDB 资源要求如下
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-deployment-private-env.md]:
```suggestion
## 部署 Cloud TiDB 资源要求
```

File: session2/chapter1/tidb-operator-deployment-private-env.md [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
+
 1. 至少 3 台服务器可供使用，服务器采用万兆网卡
 2. 至少 6 个 PV 可供使用(3 PV 用于 TiKV 实例，分布于不同服务器；3 PV用于 PD 部署，分布于不同服务器)
 
-### Linux 内核参数要求如下:
+## Linux 内核参数要求如下
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-deployment-private-env.md]:
```suggestion
## Linux 内核参数要求
```

File: session2/chapter1/tidb-operator-deployment-private-operator.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
+
+# 部署 TiDB Operator
+
+## 操作步骤如下
+
+### 一、通过 helm 安装 TiDB Operator
+
+创建 TiDB CRD
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-deployment-private-operator.md]:
```suggestion
创建 TiDB CRD：
```

File: session2/chapter1/tidb-operator-deployment-private-operator.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    customresourcedefinition.apiextensions.k8s.io/tidbmonitors.pingcap.com created
+    customresourcedefinition.apiextensions.k8s.io/tidbinitializers.pingcap.com created
+    customresourcedefinition.apiextensions.k8s.io/tidbclusterautoscalers.pingcap.com created
+    # kubectl get crd tidbclusters.pingcap.com
+    NAME                       CREATED AT
+    tidbclusters.pingcap.com   2020-03-07T09:58:09Z
+
+```
+
+下载 TiDB Operator 的 helm chart 文件
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-deployment-private-operator.md]:
```suggestion
下载 TiDB Operator 的 helm chart 文件：
```

File: session2/chapter1/tidb-operator-deployment-private-operator.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+    从 https://github.com/pingcap/tidb-operator/releases 下载 tidb-operator-chart-v1.0.6.tgz 文件放到 /root/chart/ 路径下
+
+    # cd /root/chart/ && tar xvf tidb-operator-chart-v1.0.6.tgz
+
+```
+
+将 /root/charts/tidb-operator/values.yaml 文件内的 scheduler.kubeSchedulerImageName 值修改为 registry.cn-hangzhou.aliyuncs.com/google_containers/kube-scheduler 以加快镜像拉取速度
+
+安装 TiDB Operator
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-deployment-private-operator.md]:
```suggestion
安装 TiDB Operator：
```

File: session2/chapter1/tidb-operator-deployment-private-operator.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+```
+
+    # kubectl get pods -n tidb-admin
+    NAME                                       READY   STATUS    RESTARTS   AGE
+    tidb-controller-manager-85d8d498bf-2n8km   1/1     Running   0          19s
+    tidb-scheduler-7c67d6c77b-qd54r            2/2     Running   0          19s
+
+```
+
+以上信息显示 Operator 运行正常
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-deployment-private-operator.md]:
```suggestion
以上信息显示 Operator 运行正常。
```

File: session2/chapter1/tidb-operator-deployment-private-pv.md [hunk 8]
```diff
@@ -59,4 +76,5 @@ local-pv-d4cf548e   1468Mi     RWO            Delete           Available
 local-pv-eb0e3c9f   1974Mi     RWO            Delete           Available           local-storage            2m56s
 
 ```
-以上信息说明 Local PV 已经在正常创建了。
+
+以上信息说明 Local PV 已经在正常创建了
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-deployment-private-pv.md]:
```suggestion
以上信息说明 Local PV 已经在正常创建了。
```

File: session2/chapter1/tidb-operator-deployment-private-tidb.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
+
+# 部署 TiDB 集群
+
+## 一、安装 TiDB Cluster
+
+```
+
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-deployment-private-tidb.md:16]: 这里再说一下从哪里下载 chart 吧

File: session2/chapter1/tidb-operator-deployment-private-tidb.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+    # helm install --namespace dba-test --name=test /root/charts/tidb-cluster -f /root/charts/tidb-cluster/values.yaml
+    NAME:   test
+    LAST DEPLOYED: Sat Mar  7 05:27:57 2020
+    NAMESPACE: dba-test
+    STATUS: DEPLOYED
+    ...
+
+```
+
+以上信息显示 TiDB Cluster 部署正常
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-deployment-private-tidb.md]:
```suggestion
以上信息显示 TiDB Cluster 部署正常。
```

File: session2/chapter1/tidb-operator-deployment-private-tidb.md [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+    test-pd-2                         1/1     Running   0          4m40s
+    test-tidb-0                       2/2     Running   0          2m13s
+    test-tidb-1                       2/2     Running   0          2m13s
+    test-tikv-0                       1/1     Running   0          2m45s
+    test-tikv-1                       1/1     Running   0          2m45s
+    test-tikv-2                       1/1     Running   0          2m45s
+
+```
+
+以上信息显示 TiDB Cluster 所有 Pod 全部运行正常
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-deployment-private-tidb.md]:
```suggestion
以上信息显示 TiDB Cluster 所有 Pod 全部运行正常。
```

File: session2/chapter1/tidb-operator-deployment-private-tidb.md [hunk 12]
```diff
@@ -0,0 +1,79 @@
...
+    test-pd                 ClusterIP   10.96.183.196    <none>        2379/TCP                         3m58s
+    test-pd-peer            ClusterIP   None             <none>        2380/TCP                         3m58s
+    test-prometheus         NodePort    10.107.17.45     <none>        9090:31800/TCP                   3m58s
+    test-tidb               NodePort    10.104.37.71     <none>        4000:30169/TCP,10080:30286/TCP   3m58s
+    test-tidb-peer          ClusterIP   None             <none>        10080/TCP                        90s
+    test-tikv-peer          ClusterIP   None             <none>        20160/TCP                        2m2s
+
+```
+
+找到 test-tidb 这个 Service 的 CLUSTER-IP，通过其访问 TiDB 集群
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-deployment-private-tidb.md]:
```suggestion
找到 test-tidb 这个 Service 的 CLUSTER-IP，通过其访问 TiDB 集群：
```

File: session2/chapter1/tidb-operator-deployment-private-tidb.md [hunk 13]
```diff
@@ -0,0 +1,79 @@
...
+    Oracle is a registered trademark of Oracle Corporation and/or its
+    affiliates. Other names may be trademarks of their respective owners.
+
+    Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
+
+    mysql> 
+
+```
+
+访问 TiDB 成功
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-deployment-private-tidb.md]:
```suggestion
访问 TiDB 成功。
```

File: session2/chapter1/tidb-operator-local-deployment.md [hunk 14]
```diff
@@ -1,60 +1,88 @@
...
+
 ```
+
     # kubectl cluster-info
       Kubernetes master is running at https://127.0.0.1:32771
       KubeDNS is running at https://127.0.0.1:32771/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
 ```
+
 输出以上信息，则说明 K8s 服务符合要求
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-local-deployment.md]:
```suggestion
输出以上信息，则说明 K8s 服务符合要求。
```

File: session2/chapter1/tidb-operator-local-deployment.md [hunk 15]
```diff
@@ -65,8 +93,11 @@ To start using your cluster, run:
     customresourcedefinition.apiextensions.k8s.io/tidbclusterautoscalers.pingcap.com unchanged
     NAME                       CREATED AT
     tidbclusters.pingcap.com   2020-03-06T13:38:32Z
+
 ```
+
 下载 TiDB Operator 的 helm chart 文件
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-local-deployment.md]:
```suggestion
下载 TiDB Operator 的 helm chart 文件：
```

File: session2/chapter1/tidb-operator-local-deployment.md [hunk 16]
```diff
@@ -129,12 +179,15 @@ To start using your cluster, run:
     test-tikv-0                       1/1     Running   0          6m58s
     test-tikv-1                       1/1     Running   0          6m58s
     test-tikv-2                       1/1     Running   0          6m58s
+
 ```
 
-以上信息显示 TiDB Cluster 所有 Pod 全部运行正常。
+以上信息显示 TiDB Cluster 所有 Pod 全部运行正常
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-local-deployment.md]:
```suggestion
以上信息显示 TiDB Cluster 所有 Pod 全部运行正常。
```

File: session2/chapter1/tidb-operator-local-deployment.md [hunk 17]
```diff
@@ -145,5 +198,7 @@ To start using your cluster, run:
     Copyright (c) 2000, 2018, Oracle, MariaDB Corporation Ab and others.
     Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
     MySQL [(none)]>
+
 ```
-显示以上输出显示 TiDB 集群部署成功。
+
+显示以上输出显示 TiDB 集群部署成功
```
Comment (DanielZhangQD) [inline session2/chapter1/tidb-operator-local-deployment.md]:
```suggestion
显示以上输出显示 TiDB 集群部署成功。
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by winkyao — APPROVED** [review]

**Pull request merged**